ansible/roles/collectory/templates/config/collectory-config.properties

```diff
@@ -143,11 +142,14 @@ gbifApiPassword={{ gbif_api_password | default('') }}
 gbifExportUrlBase={{ gbif_export_url_base | default('') }}
 useGbifDoi={{ gbif_use_doi | default('false') }}
 gbifRegistrationDryRun={{ gbif_registration_dry_run | default('true') }}
+
 {% if collectory_version is version('3.0.0', '<') %}
 gbifLicenceMappingUrl={{ gbif_licence_mapping_url | default('file:///data/ala-collectory/config/default-gbif-licence-mapping.json') }}
 {% else %}
 gbifLicenceMappingUrl={{ gbif_licence_mapping_url | default('file:///data/collectory/config/default-gbif-licence-mapping.json') }}
 {% endif %}
+gbifLicenceMappingUrl={{ gbif_licence_mapping_url | default('file:///data/collectory/config/default-gbif-licence-mapping.json') }}
```

Comment:
`gbifLicenceMappingUrl` is already set by the previous `if` block.

Reply:
removed